### DISCUSSION TITLE
Undo container?

### DISCUSSION THREAD

**Discussion by therealgilles** [tailwindlabs/tailwindcss#3864] [Help]
If there a way to use 'container' to the small screen size but not for the bigger ones? The documentation seems to suggest it's not possible, i.e. it's only starting from a certain breakpoint and up:

```
The container class also includes responsive variants like md:container by default that allow you to make something behave like a container at only a certain breakpoint and up
```

Suggestions welcome.

**Comment by simonswiss** [comment]
Hey!

The `container` component class is setting a responsive `max-width` property. If you want to "opt out" of this for a larger breakpoint, you can apply a `max-w-none` class like so:

```html
<div class="bg-red-200 h-20 container lg:max-w-none"></div>
```

This will apply a max width container to the div for all screen sizes until the viewport reaches the `lg` breakpoint. Past that point, the div will use the entire width available 👍

Here's a Play example:

https://play.tailwindcss.com/OFm3tt0C1r

Hope it helps!

**Reply by therealgilles** [reply]
Thank you for your response @simonswiss, much appreciated.

**Reply by therealgilles** [reply]
Follow-up question: how can I opt out for the smallest screen size or only for a specific breakpoint?

For instance, I would no padding for the smallest screen size, padding for sm/md, then then no padding for lg and up. I can't see a way to do it this way without hardcoding the padding for sm/md.

Any advice?
